src/spring/azext_spring/_params.py

```diff
@@ -1307,3 +1307,8 @@ def prepare_common_logs_argument(c):
         c.argument('zone_id', help='The resource id of the private DNS zone which you would like to configure with the service instance.')
     with self.argument_context('spring private-dns-zone clean') as c:
         c.argument('service', service_name_type)
+
+    with self.argument_context('spring export') as c:
+        c.argument('service', service_name_type)
+        c.argument('target', help='The target Azure service to migrate to, allowed values: aca, azure-container-apps.')
```

Comment:
Please define this param with `arg_type=get_enum_type(["aca", "azure-container-apps"])` if it is enumerable. After defining with that, it is not necessary to specify the available values and CLI will verify the users' input automatically.
```suggestion
        c.argument('target', arg_type=get_enum_type(["aca", "azure-container-apps"]), help='The target Azure service to migrate to.')
```